docs/en/remote-flow.rst

```diff
@@ -124,7 +124,8 @@ A non-normative example of the HTTP request is represented below:
       "request_object_signing_alg_values_supported": [
         "ES256"
       ],
-      "presentation_definition_uri_supported": false
+      "presentation_definition_uri_supported": false,
+      "client_id_schemes_supported": ["https"]
   }
```

Comment:
This non-normative example must be removed as it is not updated. The correct one is in line 251 (with `wallet_metadata` and `wallet_nonce`)